sig-scheduling/annual-report-2024.md

```diff
@@ -69,12 +72,12 @@
 ## Operational
 
 Operational tasks in [sig-governance.md]:
-- [ ] [README.md] reviewed for accuracy and updated if needed
-- [ ] [CONTRIBUTING.md] reviewed for accuracy and updated if needed
+- [x] [README.md] reviewed for accuracy and updated if needed
+- [x] [CONTRIBUTING.md] reviewed for accuracy and updated if needed
 - [ ] Other contributing docs (e.g. in devel dir or contributor guide) reviewed for accuracy and updated if needed
```

Comment:
So, actually I don't think we have paid attention enough to [these docs](https://github.com/kubernetes/community/tree/master/contributors/devel/sig-scheduling). We should review and update some docs (e.g., need to mention QHint at the queue doc etc)

Reply:
Opened https://github.com/kubernetes/community/issues/8289 to track this problem